packages/devextreme/testing/tests/DevExpress.core/utils.object.tests.js

```diff
@@ -171,7 +171,21 @@ QUnit.test('deepExtendArraySafe utility does not throw an error with \'null\' de
     assert.equal(result.deepProp.toChange, 'changed value');
 });
 
-QUnit.test('deepExtendArraySafe utility does not pollute object prototype', function(assert) {
-    objectUtils.deepExtendArraySafe({ }, JSON.parse('{ "__proto__": { "pollution": true }}'), true);
-    assert.ok(!('pollution' in { }), 'object prototype is not polluted');
+QUnit.test('deepExtendArraySafe sets undefined', function(assert) {
```

Comment:
Now in one test case, we test two scenarios:
1) The ```shouldCopyUndefined``` false case
2) The ```shouldCopyUndefined``` true case

It's better to have two different tests for two different test scenarios
So, let's split this test case into two